XLS-47d-PriceOracles/README.md

```diff
@@ -101,81 +102,58 @@ We define a new transaction **OracleSet** for creating or updating a `PriceOracl
 
 - Create or own the `Account` on the XRPL with sufficient XRP balance to meet the XRP reserve and the transaction fee requirements.
 - The Oracle Provider has to publish the `Account` account public key so that it can be used for verification by dApp’s.
-- The Oracle Provider has to publish a registry of available Price Oracles with their unique Oracle Sequence. The hash of the `Account` and the `OracleSequence` uniquely identifies the Price Oracle on-ledger object.
+- The Oracle Provider has to publish a registry of available Price Oracles with their unique `OracleDocumentID`. The hash of the `Account` and the `OracleDocumentID` uniquely identifies the Price Oracle on-ledger object.
 
-#### Transaction fields for **OracleSet** transaction
+#### Example of OracleSet transaction JSON
 
-|FieldName | Required? | JSON Type | Internal Type |
-|:---------|:-----------|:---------------|:------------|
-| `TransactionType` | :heavy_check_mark: | `string`| UINT16 |
+    {
+        "TransactionType": "OracleSet",
+        "Account": "rsA2LpzuawewSBQXkiju3YQTMzW13pAAdW",
+        "OracleDocumentID": 34,
+        "Provider": "70726F7669646572",
+        "LastUpdateTime": 743609014,
+        "AssetClass": "63757272656E6379",
+        "PriceDataSeries": [
+          {
+            "PriceData": {
+              "BaseAsset": "XRP",
+              "QuoteAsset": "USD",
+              "AssetPrice": 740,
+              "Scale": 3
+            }
+          }
+        ]
+    }
 
-Indicates a new transaction type `OracleSet`. The integer value is TBD.
+#### Transaction fields for **OracleSet** transaction
 
 |FieldName | Required? | JSON Type | Internal Type |
 |:---------|:-----------|:---------------|:------------|
+| `TransactionType` | :heavy_check_mark: | `string`| UINT16 |
 | `Account` | :heavy_check_mark: | `string` | `ACCOUNTID` |
-
-`Account` is the XRPL account that has update and delete privileges on the Oracle being set.
-
-|FieldName | Required? | JSON Type | Internal Type |
-|:---------|:-----------|:---------------|:------------|
-| `OracleSequence` | :heavy_check_mark: | `string` | `UINT32` |
-
-`OracleSequence` is a unique identifier of the Price Oracle for the given Account.
-
-|FieldName | Required? | JSON Type | Internal Type |
-|:---------|:-----------|:---------------|:------------|
+| `OracleDocumentID` | :heavy_check_mark: | `string` | `UINT32` |
 | `Provider` | | `string` | `BLOB` |
```

Comment:
What do you think of a third marker (after "no marker" and :heavy_check_mark:) to indicate a field is conditionally required, like `Provider` and `AssetClass`? Perhaps :grey_question: ?